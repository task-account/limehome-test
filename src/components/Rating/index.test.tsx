import renderer from "react-test-renderer";
import { Rating } from ".";

jest.mock("@utils/hooks", () => ({
  withStyles: () => () => ({ useStyles: jest.fn() })
}));

describe(`Rating`, () => {
  it("render", () => {
    const tree = renderer.create(<Rating rating={"2.3"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
