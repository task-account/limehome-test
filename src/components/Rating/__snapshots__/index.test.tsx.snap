// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating render 1`] = `
<View>
  <Text>
    2.3
  </Text>
  <View
    style={
      Object {
        "flex": undefined,
        "marginBottom": undefined,
        "marginHorizontal": 2,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "marginVertical": undefined,
      }
    }
  />
  <Text />
</View>
`;
